visual-js/visual-snapshots/src/app/pdf-file.ts

```diff
@@ -1,4 +1,5 @@
-export interface PdfFile {
+export interface LoadedPdfFile {
```

Comment:
At this point we don't care if this is a PDF file or now. For us, this is simply an image

Reply:
Not really - this has pages in it. We need to have the option to load a particular page in workers.

Reply:
Okey - a set of images then. Or a document. Not necessarily a PDF file.
I strongly believe an abstraction would be beneficial here. I always prefer to make application easily extendable. In this particular application it will never be an issue though so I don't think of this a a blocker.

Reply:
Let's explore these changes in a separate pull request.